src/coreclr/build-test.sh

```diff
@@ -5,6 +5,12 @@ build_test_wrappers()
     if [[ "$__BuildTestWrappers" -ne -0 ]]; then
         echo "${__MsgPrefix}Creating test wrappers..."
 
+        if [[ $__Mono -eq 1 ]]; then
+            export RuntimeName="mono"
+        else
+            export RuntimeName="coreclr"
+        fi
```

Comment:
We should reuse the `RuntimeFlavor` property we already use in other parts of the build to decide between Mono/CoreCLR instead of inventing another property here. Can be done in a follow-up PR though :)

Reply:
Agree. If we're doing it as a follow up PR though we should file an issue to track the work so it doesn't get lost.